src/index.js

```diff
@@ -57,14 +65,20 @@ const getTextNodesInElementContainingText = (element, text) => {
 const getRangesForSearchTermInElement = (element, search) => {
   const ranges = [];
   const lowerCaseSearch = search.toLowerCase();
-  if (!element.firstChild) return ranges;
-  const text = element.textContent?.toLowerCase() || "";
+  if (!element.childNodes.length === 0) return ranges;
```

Comment:
I don't understand this change. The initial `!` makes it the opposite of the previous test.

Reply:
Fixed